common/types/loadbalancer.go

```diff
@@ -47,6 +48,10 @@ type LBBackEnd struct {
 	Weight uint16
 }
 
+func (lbbe *LBBackEnd) String() string {
+	return fmt.Sprintf("%s, weight: %d", lbbe.L3n4Addr.String(), lbbe.Weight)
```

Comment:
Isn't there a String() already for this? I was 80% sure there was.

Reply:
Since `LBBackEnd` is embedded with an `L3n4Addr`, I guess we could call `String()` on it, and it would use the `L3n4Addr.String()` function, but we wouldn't get the value for the weight for the `LBBackEnd`.

Reply:
I mean for LBBackend. It's fine now